app/style/app.css

```diff
@@ -9,14 +9,7 @@ gaia-modal {
   background: white;
 }
 
-.settings gaia-list gaia-button {
-  /* XXX/drs: Why do we have to make this !important? It's very specific. */
-  margin: 0 0 0 auto !important;
-
-  --button-background: red;
-  text-align: center;
-}
-
-.settings gaia-list .addon-time {
+.settings gaia-list p,
+.settings gaia-list i {
```

Comment:
Does bug 1150106 (https://bugzilla.mozilla.org/show_bug.cgi?id=1150106) fix add-on CSS loading? If not, we should file another bug for that.